src/periphery/StrategyAprOracle.sol

```diff
@@ -46,12 +46,12 @@ contract StrategyAprOracle {
             uint256 unbacked,
             ,
             ,
-            uint256 totalStableDebt,
+            ,
```

Comment:
`protocolDataProvider` is not upgradable contract: https://etherscan.io/address/0x7B4EB56E7CD4b454BA8ff71E4518426369a138a3#code

Probably you will need to change the address also to use the new response.